src/web/tex.rkt

```diff
@@ -1,8 +1,6 @@
 #lang racket
-(require "../common.rkt" "../programs.rkt" "../syntax/types.rkt" "../float.rkt" "../interface.rkt")
 
-
-(provide js-tex-include texify-expr texify-prog)
+(provide js-tex-include)
 
 (define js-tex-include
```

Comment:
We should move this to `src/web/common.rkt` and delete the `src/web/tex.rkt` file.